src/pages/solution/adobe-campaign-classic/push-templates/tabs/basic.md

```diff
@@ -64,6 +65,7 @@ Below is a sample of what a payload might look like for a basic notification:
                 "adb_title": "game request",
 				"adb_body": "shall we play a game?",
 				"adb_sound": "bingBong",
+				"adb_icon": "ic_knight",
```

Comment:
if we ignore `adb_icon` when `adb_small_icon` is present, does it make sense to include both in the example?

Reply:
good point, i think we can leave `adb_icon` out of the example